### ISSUE TITLE
Color contrast issue on uipress website

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#8571]
Siteimprove is reporting Color contrast does not meet minimum requirement on multiple pages of uipress.uiowa.edu.

A couple examples:
https://my2.siteimprove.com/Inspector/949257/A11Y/Page?pageId=45742242161&impmd=264ED37B65FA6186129334A508391970#/sia-r69/failed/ZGhlYTVZSU0=;YTAxMzMyYzI=

https://my2.siteimprove.com/Inspector/949257/A11Y/Page?pageId=45742247233&impmd=264ED37B65FA6186129334A508391970#/sia-r69/failed/bUdyV0RkQkk=;ZDExYjQ5NzY=

https://my2.siteimprove.com/Inspector/949257/A11Y/Page?pageId=45742247598&impmd=264ED37B65FA6186129334A508391970#/sia-r69/failed/OW5vdmdzTlI=;ZjQzZDRjYjg=

We should increase the color contrast to 4.5:1 or higher where we are using this link color.

**Comment by bspeare** [comment]
This is similar to https://github.com/uiowa/uiowa/pull/8496, we need to change the linked author's name from using the `subtitle` to the `card__meta`.
